CMakeLists.txt

```diff
@@ -21,13 +21,19 @@ endif()
 
 find_package(HPX REQUIRED)
 find_package(VTK REQUIRED)
+
 include(${VTK_USE_FILE})
 
 add_library(LibsModule 
     ./include/writer.cpp
 )
 target_link_libraries(LibsModule ${VTK_LIBRARIES})
 
+include_directories("/usr/local/include")
```

Comment:
I do not like the fixed path there. Is there a FindCMake available for gmsh and metis? 

This will only work, if one installed both libs as root user. However, on a cluster, we have to install them in our home or scratch and this will not work? 

We have three alternatives here:

1. Use the FindCmake provided by the project

2.  Provide two CMake variables, e.g. GMSH_ROOT and METIS_ROOT, and the user provides the path to the both libs and you add /lib and /include

3) Write your own FindCmake

Please find one example here

https://github.com/STEllAR-GROUP/octotiger/blob/master/cmake/FindSilo.cmake

Reply:
Ok done Patrick. I have done the option 2. Provided CMake variables GMSH_DIR and METIS_DIR.
I have used *_DIR because previously I had used VTK_DIR for vtk library.
So instead of using *_ROOT i have kept is as *_DIR.
Let me know if you want me to change that